### PR TITLE
feat(GlyphsBlock): adjust border, radius & settings

### PR DESCRIPTION
**Pull request by Walkeezy**
Fixes for design review: https://app.clickup.com/t/863ghy2bx

### CODE REVIEW

File: packages/glyphs-block/src/GlyphsBlock.tsx [hunk 1]
```diff
@@ -40,7 +46,18 @@ export const GlyphsBlock = ({ appBridge }: BlockProps): ReactElement => {
...
+                    }),
+                    ...(isSixthChar && {
+                        borderTopRightRadius: getRadiusValue(radiusChoice, hasRadius, radiusValue),
+                    }),
+                    ...(isBottomLeftChar && {
+                        borderBottomLeftRadius: getRadiusValue(radiusChoice, hasRadius, radiusValue),
+                    }),
+                    ...(isBottomRightChar && {
+                        borderBottomRightRadius: getRadiusValue(radiusChoice, hasRadius, radiusValue),
+                    }),
```
Comment (celine-s) [inline packages/glyphs-block/src/GlyphsBlock.tsx:60]: this now gets a bit big :D. you could use `Object.assign()`. Wdyt?

Like: 
const style = Object.assign(
            {},
            hasBackground && { backgroundColor: toRgbaString(backgroundColor) },
...
}

File: packages/glyphs-block/src/GlyphsBlock.tsx [hunk 2]
```diff
@@ -40,7 +46,18 @@ export const GlyphsBlock = ({ appBridge }: BlockProps): ReactElement => {
                         outlineWidth: borderWidth,
```
Comment (celine-s) [inline packages/glyphs-block/src/GlyphsBlock.tsx:46]: you could take all the border infos into one css: `outline: `${borderWidth} ${borderStyle} ${toRgbaString(borderColor)}`,`

Comment (Walkeezy) [reply packages/glyphs-block/src/GlyphsBlock.tsx]: Changed 👍🏻

File: packages/glyphs-block/src/GlyphsBlock.tsx (hunk 1 above)
Comment (Walkeezy) [reply packages/glyphs-block/src/GlyphsBlock.tsx]: I think it's easier understandable without Object.assign(), but moved the styles to a const to make it a bit cleaner.

**Review by celine-s — APPROVED** [review]
🚀

**Review by ragi96 — APPROVED** [review]
🚀

**Pull request merged**